src/main/java/qna/domain/Answer.java

```diff
@@ -3,91 +3,101 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.*;
 import java.util.Objects;
 
-public class Answer {
-    private Long id;
-    private Long writerId;
-    private Long questionId;
-    private String contents;
-    private boolean deleted = false;
-
-    public Answer(User writer, Question question, String contents) {
-        this(null, writer, question, contents);
-    }
-
-    public Answer(Long id, User writer, Question question, String contents) {
-        this.id = id;
-
-        if (Objects.isNull(writer)) {
-            throw new UnAuthorizedException();
-        }
-
-        if (Objects.isNull(question)) {
-            throw new NotFoundException();
-        }
-
-        this.writerId = writer.getId();
-        this.questionId = question.getId();
-        this.contents = contents;
-    }
-
-    public boolean isOwner(User writer) {
-        return this.writerId.equals(writer.getId());
+@Entity
+public class Answer extends BaseTimeEntity {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(nullable = false, updatable = false)
+  private Long id;
+  private Long writerId;
+  private Long questionId;
+  private String contents;
+  @Column(nullable = false)
+  private boolean deleted = false;
```

Comment:
```suggestion
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(nullable = false, updatable = false)
  private Long id;
  
  private Long writerId;
  
  private Long questionId;
  
  private String contents;
  
  @Column(nullable = false)
  private boolean deleted = false;
```
가독성을 위해 적절히 공백을 추가해 주세요 :)

Reply:
앗 넵 확실히 공백을 추가하는 것이 가독성에 좋네요!! 조언 감사합니다!